### PR TITLE
DATAGEODE-387 - Implement CrudRepository.delete(Iterable<ID> ids).

### PR DESCRIPTION
**Pull request by schauder**
Merge after DATACMNS-800

### CODE REVIEW

File: spring-data-geode/src/main/java/org/springframework/data/gemfire/repository/support/SimpleGemfireRepository.java [hunk 1]
```diff
@@ -310,6 +311,11 @@ public void deleteAll(@NonNull Iterable<? extends T> entities) {
 		CollectionUtils.nullSafeIterable(entities).forEach(this::delete);
 	}
 
+	@Override
+	public void deleteAllById(Iterable<? extends ID> ids) {
+		CollectionUtils.nullSafeIterable(ids).forEach(this::deleteById);
```
Comment (mp911de) [inline spring-data-geode/src/main/java/org/springframework/data/gemfire/repository/support/SimpleGemfireRepository.java:316]: @jxblum does it make sense to expose `removeAll(…)` on the Template API?

File: spring-data-geode/src/test/java/org/springframework/data/gemfire/repository/support/SimpleGemfireRepositoryIntegrationTests.java [hunk 2]
```diff
@@ -15,24 +15,14 @@
  */
 package org.springframework.data.gemfire.repository.support;
 
-import static org.assertj.core.api.Assertions.assertThat;
```
Comment (mp911de) [inline spring-data-geode/src/test/java/org/springframework/data/gemfire/repository/support/SimpleGemfireRepositoryIntegrationTests.java:18]: We should verify where this formatting change comes from.

Comment (schauder) [reply spring-data-geode/src/test/java/org/springframework/data/gemfire/repository/support/SimpleGemfireRepositoryIntegrationTests.java:18]: Another case of not configured formatter. Fixed.

**Comment by mp911de** [comment]
That's merged and polished now.

**Pull request closed without merging**